packages/wallet/backend/src/app.ts

```diff
@@ -269,4 +273,24 @@ export class App {
 
     return app
   }
+
+  private async processPendingTransactions() {
+    const transactionService = await this.container.resolve(
+      'transactionService'
+    )
+    return transactionService
+      .processPendingIncomingPayments()
+      .catch(() => true)
+      .then((trx) => {
+        if (trx) {
+          process.nextTick(() => this.processPendingTransactions())
+        } else {
+          setTimeout(() => this.processPendingTransactions(), 5000).unref()
```

Comment:
I think the interval is too small, it is not as important to have the status updated as it does not influence at anything so it easily can be 1 or even 5 mins